### PR TITLE
Device config added for Bitron AV2010/26 wireless socket and brightne…

### PR DESCRIPTION
**Pull request by the-guitarman**
…ss regulator

The Bitron AV2010/26 device is now supported to switch the "state" "ON" or "OFF". 
I did not test the brightness change.

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -8410,6 +8410,16 @@ const devices = [
...
     },
+    {
+        zigbeeModel: ['902010/26'],
+        model: 'AV2010/26',
+        vendor: 'Bitron',
+        description: 'Wireless socket and brightness regulator',
+        fromZigbee: [fz.on_off, fz.brightness],
+        toZigbee: [tz.light_onoff_brightness],
+        exposes: [e.switch()],
+        meta: {configureKey: 1},
```
Comment (Koenkk) [inline devices.js:8421]: - Remove the lines with `fromZigbee`, toZigbee` and `exposes` and add `extend: preset.light_onoff_brightness()`.
- Do you get state feedback when clicking the button on the device itself?

Comment (the-guitarman) [reply devices.js:8419]: - I changed the lines how you suggested.
- No, I don't get any feedback when I click the button in the device itself. I tested it several times and experimented with different configurations. I'm able to switch the device on or off by publishing the message `{"state": "ON"}` or `{"state": "OFF"}` to `zigbee2mqtt/FRIENDLY_NAME/set` only.

Comment (Koenkk) [reply devices.js]: Update your pr, can you check if it works with this config?

Comment (the-guitarman) [reply devices.js]: I tested your update. If I start my zigbee2mqtt service I get this error:
```bash
Zigbee2MQTT:error 2021-02-04 11:15:16: Failed to configure 'bitron_home_201026_01', attempt 1 (Error: Bind 0x00124b0008a65bb9/1 genOnOff from '0x00124b001938c297/1' failed (AREQ - ZDO - bindRsp after 10000ms)
    at Timeout._onTimeout (/home/myusername/repos/zigbee2mqtt/node_modules/zigbee-herdsman/dist/utils/waitress.js:46:35)
    at listOnTimeout (node:internal/timers:556:17)
    at processTimers (node:internal/timers:499:7))
Zigbee2MQTT:info  2021-02-04 11:15:16: Configuring 'bitron_home_201026_02'
Zigbee2MQTT:error 2021-02-04 11:15:26: Failed to configure 'bitron_home_201026_02', attempt 1 (Error: Bind 0x00124b0008a69451/1 genOnOff from '0x00124b001938c297/1' failed (AREQ - ZDO - bindRsp after 10000ms)
    at Timeout._onTimeout (/home/myusername/repos/zigbee2mqtt/node_modules/zigbee-herdsman/dist/utils/waitress.js:46:35)
    at listOnTimeout (node:internal/timers:556:17)
    at processTimers (node:internal/timers:499:7))
```
bitron_home_201026_01 and bitron_home_201026_02 are the friendly names of my two devices.

Comment (Koenkk) [reply devices.js]: Can you provide the data/database.db of this device?

Comment (the-guitarman) [reply devices.js]: And I tested the device button and again no feedback.

Comment (the-guitarman) [reply devices.js]: Here are the data/database.db entries of both 201026 devices:
```json
{"id":5,"type":"Router","ieeeAddr":"0x00124b0008a65bb9","nwkAddr":11924,"manufId":0,"manufName":"Bitron Home","powerSource":"Mains (single phase)","modelId":"902010/26","epList":[1,4],"endpoints":{"1":{"profId":260,"epId":1,"devId":257,"inClusterList":[0,3,4,5,6,8],"outClusterList":[0],"clusters":{"genBasic":{"attributes":{"modelId":"902010/26","manufacturerName":"Bitron Home","powerSource":1,"zclVersion":1,"appVersion":15,"hwVersion":0,"dateCode":"20151005        "}}},"binds":[],"configuredReportings":[],"meta":{}},"4":{"profId":260,"epId":4,"devId":257,"inClusterList":[],"outClusterList":[25],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":15,"hwVersion":0,"dateCode":"20151005        ","zclVersion":1,"interviewCompleted":true,"meta":{"configured":1},"lastSeen":1612361153752}
{"id":6,"type":"Router","ieeeAddr":"0x00124b0008a69451","nwkAddr":54044,"manufId":0,"manufName":"Bitron Home","powerSource":"Mains (single phase)","modelId":"902010/26","epList":[1,4],"endpoints":{"1":{"profId":260,"epId":1,"devId":257,"inClusterList":[0,3,4,5,6,8],"outClusterList":[0],"clusters":{"genBasic":{"attributes":{"modelId":"902010/26","manufacturerName":"Bitron Home","powerSource":1,"zclVersion":1,"appVersion":15,"hwVersion":0,"dateCode":"20151005        "}}},"binds":[],"configuredReportings":[],"meta":{}},"4":{"profId":260,"epId":4,"devId":257,"inClusterList":[],"outClusterList":[25],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":15,"hwVersion":0,"dateCode":"20151005        ","zclVersion":1,"interviewCompleted":true,"meta":{"configured":1},"lastSeen":1612382490954}
```

Comment (Koenkk) [reply devices.js]: Can you try removing and repairing the device? https://www.zigbee2mqtt.io/information/mqtt_topics_and_message_structure.html#zigbee2mqttbridgerequestdeviceremove I don't understand why it doesn't work as this also works for other Bitron devices.

Comment (the-guitarman) [reply devices.js]: I tried to remove it, but it does not work:
```txt
Zigbee2MQTT:debug 2021-02-04 21:24:58: Received MQTT message on 'zigbee2mqtt/bridge/request/device/remove' with data '{"id": "bitron_home_201026_02"}'
Zigbee2MQTT:info  2021-02-04 21:24:58: Removing device 'bitron_home_201026_02' (block: false, force: false)
Zigbee2MQTT:error 2021-02-04 21:25:08: Request 'zigbee2mqtt/bridge/request/device/remove' failed with error: 'Failed to remove device 'bitron_home_201026_02' (block: false, force: false) (Error: AREQ - ZDO - mgmtLeaveRsp after 10000ms)'
Zigbee2MQTT:info  2021-02-04 21:25:08: MQTT publish: topic 'zigbee2mqtt/bridge/response/device/remove', payload '{"data":{},"error":"Failed to remove device 'bitron_home_201026_02' (block: false, force: false) (Error: AREQ - ZDO - mgmtLeaveRsp after 10000ms)","status":"error"}'
```

Comment (Koenkk) [reply devices.js]: Use force remove (also documented here: https://www.zigbee2mqtt.io/information/mqtt_topics_and_message_structure.html#zigbee2mqttbridgerequestdeviceremove)

Comment (the-guitarman) [reply devices.js]: Ok, I did a force remove than I paired it again. Looks interesting now:

```txt
Zigbee2MQTT:info  2021-02-05 20:08:38: Device '0x00124b0008a69451' joined
Zigbee2MQTT:info  2021-02-05 20:08:38: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x00124b0008a69451","ieee_address":"0x00124b0008a69451"},"type":"device_joined"}'
Zigbee2MQTT:info  2021-02-05 20:08:38: Starting interview of '0x00124b0008a69451'
Zigbee2MQTT:info  2021-02-05 20:08:38: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x00124b0008a69451","ieee_address":"0x00124b0008a69451","status":"started"},"type":"device_interview"}'
Zigbee2MQTT:info  2021-02-05 20:08:38: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0x00124b0008a69451"},"type":"device_connected"}'
Zigbee2MQTT:info  2021-02-05 20:08:38: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0x00124b0008a69451"},"type":"pairing"}'
Zigbee2MQTT:debug 2021-02-05 20:08:39: Device '0x00124b0008a69451' announced itself
Zigbee2MQTT:info  2021-02-05 20:08:39: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x00124b0008a69451","ieee_address":"0x00124b0008a69451"},"type":"device_announce"}'
Zigbee2MQTT:info  2021-02-05 20:08:39: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"0x00124b0008a69451"},"type":"device_announced"}'
Zigbee2MQTT:debug 2021-02-05 20:08:39: Device '0x00124b0008a69451' announced itself
Zigbee2MQTT:info  2021-02-05 20:08:39: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x00124b0008a69451","ieee_address":"0x00124b0008a69451"},"type":"device_announce"}'
Zigbee2MQTT:info  2021-02-05 20:08:39: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"0x00124b0008a69451"},"type":"device_announced"}'
Zigbee2MQTT:debug 2021-02-05 20:09:03: Received Zigbee message from '0x00124b0008a69451', type 'readResponse', cluster 'genBasic', data '{"modelId":"902010/26"}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2021-02-05 20:09:04: Received Zigbee message from '0x00124b0008a69451', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":"Bitron Home"}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2021-02-05 20:09:04: Received Zigbee message from '0x00124b0008a69451', type 'readResponse', cluster 'genBasic', data '{"powerSource":1}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2021-02-05 20:09:04: Received Zigbee message from '0x00124b0008a69451', type 'readResponse', cluster 'genBasic', data '{"zclVersion":1}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2021-02-05 20:09:05: Received Zigbee message from '0x00124b0008a69451', type 'readResponse', cluster 'genBasic', data '{"appVersion":15}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2021-02-05 20:09:05: Received Zigbee message from '0x00124b0008a69451', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2021-02-05 20:09:05: Received Zigbee message from '0x00124b0008a69451', type 'readResponse', cluster 'genBasic', data '{"hwVersion":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2021-02-05 20:09:05: Received Zigbee message from '0x00124b0008a69451', type 'readResponse', cluster 'genBasic', data '{"dateCode":"20151005        "}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2021-02-05 20:09:05: Received Zigbee message from '0x00124b0008a69451', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2021-02-05 20:09:05: Successfully interviewed '0x00124b0008a69451', device has successfully been paired
Zigbee2MQTT:info  2021-02-05 20:09:05: Device '0x00124b0008a69451' is supported, identified as: Bitron Wireless socket and brightness regulator (AV2010/26)
Zigbee2MQTT:info  2021-02-05 20:09:05: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":{"description":"Wireless socket and brightness regulator","exposes":[{"features":[{"access":7,"description":"On/off state of this light","name":"state","property":"state","type":"binary","value_off":"OFF","value_on":"ON","value_toggle":"TOGGLE"},{"access":7,"description":"Brightness of this light","name":"brightness","property":"brightness","type":"numeric","value_max":254,"value_min":0}],"type":"light"},{"access":2,"description":"Triggers an effect on the light (e.g. make light blink for a few seconds)","name":"effect","property":"effect","type":"enum","values":["blink","breathe","okay","channel_change","finish_effect","stop_effect"]},{"access":1,"description":"Link quality (signal strength)","name":"linkquality","property":"linkquality","type":"numeric","unit":"lqi","value_max":255,"value_min":0}],"model":"AV2010/26","supports_ota":false,"vendor":"Bitron"},"friendly_name":"0x00124b0008a69451","ieee_address":"0x00124b0008a69451","status":"successful","supported":true},"type":"device_interview"}'
```

Comment (the-guitarman) [reply devices.js]: Oh, I get feedback now when I click the button at the device: 

```txt
Zigbee2MQTT:debug 2021-02-05 21:23:16: Received Zigbee message from '0x00124b0008a69451', type 'attributeReport', cluster 'genOnOff', data '{"onOff":1}' from endpoint 1 with groupID 0
Zigbee2MQTT:info  2021-02-05 21:23:16: MQTT publish: topic 'zigbee2mqtt/0x00124b0008a69451', payload '{"linkquality":23,"state":"ON"}'
```

Comment (Koenkk) [reply devices.js]: Great, that means the configure() doesn't do anything so I removed it.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**